extensions/amp-story/amp-story.md

```diff
@@ -1386,6 +1386,177 @@ The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON confi
   ]
 }
 
+```
+## Experimental Feature: Branching
+
+Branching is a set of experimental features that allow the construction of non-linear stories. By utilizing these features, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story.  Example use cases include quizzes, choose-your-own-story experiences, and story table of contents.
+
+All branching features are behind an `amp-story-branching` experiment. To turn on the experiment, enable it in your dev console:
+
+``` AMP.toggleExperiment('amp-story-branching') ```
+
+For more information see, [experimental features](https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html?format=stories).
+
+## Manipulating Navigation
+The features listed below allow you to manipulate navigation within a story. When using branching features, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. That is, if a user navigates from page-1 to page-5 and then page-6, navigating backwards will follow this exact path (page-6 to page-5 and finally page-1). The other pages will not be exposed to the user.
+
+### `goToPage` action
+The `goToPage` action allows for 'jumping' from one page within a story to another. The action can be used with allowed elements in `amp-story` and their respective events. Clicking on an element with this action should bring the user to another page within the same story. Using this action **requires** that you give the `amp-story` element an id, a unique identifier, as the story itself is the target.  Read more about actions and events in AMP [here](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.html?format=stories).
```

Comment:
This says give the `amp-story` element the id - but the example looks like the id is on `amp-story-page`?

Reply:
The `amp-story-page` elements in the code snippet all have IDs, but its parent `amp-story` has the id `story`! It might be confusing since everything has an ID here, but I don't think we're able to bold any part of a code snippet to make it clearer.